### ISSUE TITLE
Layers in Folly?

### ISSUE THREAD

**Issue by BrianErikson** [7yl4r/LifeGenes#23]
Are layers really necessary at this point? Since we're not limited by Golly anymore, we could just have one 'layer' and each cell could just carry their own color value. Normally I'd just make the change but since this vastly changes the way the engine would work I thought I'd ask.

**Comment by 7yl4r** [comment]
You're right. Layers are just for golly really. I've had some ideas about
having layers to represent overlapping things in the environment... E.g. a
physical layer, energy later, chemical signal layer, and so on... But those
ideas are so far off it's probably better to just drop the layer business
now if it makes things easier.

**Comment by BrianErikson** [comment]
I think we can leave the groundwork for layers alone in that case, and just leave it open for future development just in case. We'll work on the 'visible' layer right now only, and once that's done you might be able to flip through layers to see different things going on with the cells. It would be nice to have the ability to add that easily if we chose to.
